recipes/edyn/all/conanfile.py

```diff
@@ -1,9 +1,10 @@
-from conans import CMake, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
-import functools
+from conan import ConanFile
+from conan.tools import files, build, scm, layout
```

Comment:
we prefer going down one more level for the imports

Reply:
I did notice that, I'll change them to one more level for consistency!